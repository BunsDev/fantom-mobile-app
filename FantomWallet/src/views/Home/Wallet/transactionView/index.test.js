import React from 'react';
import { shallow } from 'enzyme';

// eslint-disable-next-line import/first
import { TransactionView } from './index';

test('renders correctly', () => {
  const wrapper = shallow(<TransactionView publicKey="0x00004" isLoading />);
  expect(wrapper).toMatchSnapshot();
});
