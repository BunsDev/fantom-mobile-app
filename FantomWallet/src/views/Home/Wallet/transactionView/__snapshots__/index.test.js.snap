// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly 1`] = `
<View
  style={
    Object {
      "flex": 1,
    }
  }
>
  0
  <View
    style={
      Object {
        "alignItems": "flex-end",
        "flex": 1,
        "paddingRight": 16,
      }
    }
  >
    <TouchableOpacity
      activeOpacity={0.2}
      onPress={[Function]}
    >
      <Image
        source={
          Object {
            "testUri": "../../../src/images/arrow_With_bar.png",
          }
        }
        style={
          Object {
            "height": 23,
            "marginTop": 16,
            "tintColor": "grey",
            "width": 25,
          }
        }
      />
    </TouchableOpacity>
  </View>
  <View
    style=""
  />
</View>
`;
